runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -110,6 +110,7 @@
 	private static final ImmutableList<String> PET_MESSAGES = ImmutableList.of("You have a funny feeling like you're being followed",
 		"You feel something weird sneaking into your backpack",
 		"You have a funny feeling like you would have been followed");
+	private static final Pattern BA_HIGH_GAMBLE_REWARD_PATTERN = Pattern.compile("(.+)!<br>High level gamble count: <col=7f0000>(.+)</col>");
```

Comment:
What is an example of text you'd receive when doing a high gamble? That should be added to this plugin's test cases.